source/_includes/_general/what-is-ce.rst

```diff
@@ -11,7 +11,8 @@ software, which consists of
 - File Management System including collaborative editing and document
   sharing
 - Tasks
-- Chat (upcoming)
+- Chat
+- Video Meeting
```

Comment:
Can you put these two lines (14-15) on the same line, similar to https://github.com/zextras/tech-doc/blob/devel/source/_includes/_general/what-is-cb.rst ?

Reply:
Hi,
Thanks for the guidance.
I have put those two lines.
Please let me know if you find them in order.
Regards